### DISCUSSION TITLE
Possible to email block whole TLD?

### DISCUSSION THREAD

**Discussion by lifeofguenter** [mastodon/mastodon#23005] [Q&A]
I would like to block complete tlds such as tk, cf and ml from signing up - as these are domains that can be registered freely.

Is that currently possible?

**Comment by mjankowski** [comment]
I believe you can ... I think the email domain block will save that value, and then the checking logic essentially works from specific to general, so if you blocked an entire TLD, that would block all domains from that TLD.
